tests/integration/targets/nxos_vrf_address_family/tasks/cli.yaml

```diff
@@ -13,6 +13,7 @@
   delegate_to: localhost
 
 - name: Run test case (connection=ansible.netcommon.network_cli)
+  when: not nxos_skip_marked | default(false)
```

Comment:
If we're not running the tests at all, the above collection tasks shouldn't be run as well. Can't we add this conditional to tasks/main.yml, which is the entry point?

Reply:
Done!